modin/_compat/pandas_api/latest/dataframe.py

```diff
@@ -0,0 +1,305 @@
+        columns=None,
+        dtype=None,
+        copy=None,
+        query_compiler=None,
+    ):
+        self._init(
+            data=data,
+            index=index,
+            columns=columns,
+            dtype=dtype,
+            copy=copy,
+            query_compiler=query_compiler,
+        )
+
+    def applymap(
+        self, func, na_action: Optional[str] = None, **kwargs
+    ):  # noqa: PR01, RT01, D200
+        return self._applymap(func, na_action=na_action, **kwargs)
+
+    def apply(
+        self, func, axis=0, raw=False, result_type=None, args=(), **kwargs
+    ):  # noqa: PR01, RT01, D200
+        return self._apply(
+            func, axis=axis, raw=raw, result_type=result_type, args=args, **kwargs
+        )
+
+    def info(
+        self,
+        verbose: Optional[bool] = None,
+        buf: Optional[IO[str]] = None,
+        max_cols: Optional[int] = None,
+        memory_usage: Optional[Union[bool, str]] = None,
+        show_counts: Optional[bool] = None,
+        null_counts: Optional[bool] = None,
+    ):  # noqa: PR01, D200
+        return self._info(
+            verbose=verbose,
+            buf=buf,
+            max_cols=max_cols,
+            memory_usage=memory_usage,
+            show_counts=show_counts,
+            null_counts=null_counts,
+        )
+
+    def pivot_table(
+        self,
+        values=None,
+        index=None,
+        columns=None,
+        aggfunc="mean",
+        fill_value=None,
+        margins=False,
+        dropna=True,
+        margins_name="All",
+        observed=False,
+        sort=True,
+    ):  # noqa: PR01, RT01, D200
+        return self._pivot_table(
+            values=values,
+            index=index,
+            columns=columns,
+            aggfunc=aggfunc,
+            fill_value=fill_value,
+            margins=margins,
+            dropna=dropna,
+            margins_name=margins_name,
+            observed=observed,
+            sort=sort,
+        )
+
+    def prod(
+        self,
+        axis=None,
+        skipna=True,
+        level=None,
+        numeric_only=None,
+        min_count=0,
+        **kwargs,
+    ):  # noqa: PR01, RT01, D200
+        validate_bool_kwarg(skipna, "skipna", none_allowed=False)
+        return self._prod(
+            axis=axis,
+            skipna=skipna,
+            level=level,
+            numeric_only=numeric_only,
+            min_count=min_count,
+            **kwargs,
+        )
+
+    def reindex(
+        self,
+        labels=None,
+        index=None,
+        columns=None,
+        axis=None,
+        method=None,
+        copy=True,
+        level=None,
+        fill_value=nan,
+        limit=None,
+        tolerance=None,
+    ):  # noqa: PR01, RT01, D200
+        axis = self._get_axis_number(axis)
+        if axis == 0 and labels is not None:
+            index = labels
+        elif labels is not None:
+            columns = labels
+        return super(LatestCompatibleDataFrame, self).reindex(
```

Comment:
Why does `reindex` not call `self._reindex`?

Reply:
It eventually will, as this `super().reindex()` would call into `LatestCompatibleBasePandasDataset.reindex()` which in turn would call `self._reindex()`